nntrainer/graph/network_graph.cpp

```diff
@@ -324,27 +324,15 @@ void NetworkGraph::setOutputLayers() {
       }
     }
 
-    if (layer_idx->getOutputDimensions().size() != layer_idx->getNumOutputs()) {
-      if (layer_idx->getNumOutputs() == 0) {
-        /** No output layer inplies its the last layer */
-        layer_idx->setOutputLayers({"__exit__"});
-        last_layer_count += 1;
-      } else {
-        /** error for any other layer */
-        throw std::logic_error("Graph node has fewer edges than expected.");
-      }
+    if (layer_idx->getNumOutputs() == 0) {
+      last_layer_count += 1;
```

Comment:
Q. now, we don't have `__exit__` anymore?

Reply:
yes, __exit__ and __data__ is not needed anymore.